### PR TITLE
10.0 mig partner identification gln mgs

### PR DESCRIPTION
**Pull request by GillesTephaneMeyomesse**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@GillesTephaneMeyomesse Can you check Codecoverage and add test for False values ?

**Comment by GillesTephaneMeyomesse** [comment]
@rousseldenis Of course I'll inform you at the next commit

**Comment by GillesTephaneMeyomesse** [comment]
@rousseldenis can you review changes

File: partner_identification_gln/README.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+============
+
+This module depends on 'stdnum' python module
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/repo/github-com-oca-partner-contact-134
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (rousseldenis) [inline partner_identification_gln/README.rst]: @GillesTephaneMeyomesse You can remove this line

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Minor comment. LGTM

**Comment by GillesTephaneMeyomesse** [comment]
@rousseldenis thank's fixed

**Review by rousseldenis — APPROVED** [review]
:+1:

**Comment by rousseldenis** [comment]
@Cedric-Pigeon

**Comment by rvalyi** [comment]
superseded by this evil PR https://github.com/OCA/partner-contact/pull/666

**Pull request closed without merging**